### PR TITLE
Fix Storage Config

### PR DESCRIPTION
**Pull request by bernardoVale**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no|yes
| API breaks?     | no|yes
| Deprecations?   | no|yes
| Related tickets | fixes #X, partially #Y, mentioned in #Z
| License         | Apache 2.0

### What's in this PR?
This PR fix the configuration of kafka logs. 

### Why?
The current implementation is not configuring the `kafka.logs` property making the cluster use the default directory which is `/tmp/kafka-logs`. The line added was borrowed from master branch

### Checklist

- [ ] Implementation tested
- [ ] Error handling code meets the [guideline](https://github.com/banzaicloud/pipeline/blob/master/docs/error-handling-guide.md)
- [ ] Logging code meets the guideline
- [ ] User guide and development docs updated (if needed)

### To Do

- [ ] If the PR is not complete but you want to discuss the approach, list what remains to be done here

### CODE REVIEW

File: pkg/resources/kafka/configmap.go [hunk 1]
```diff
@@ -55,7 +55,9 @@ metric.reporters=com.linkedin.kafka.cruisecontrol.metricsreporter.CruiseControlM
 cruise.control.metrics.reporter.bootstrap.servers={{ .CruiseControlBootstrapServers }}
 broker.id={{ .Id }}
 
-{{ .StorageConfig }}
+{{ if .StorageConfig }}
```
Comment (baluchicken) [inline pkg/resources/kafka/configmap.go]: I would like to leave the {{ . StorageConfig }} and add the fix to the generator function.
https://github.com/banzaicloud/kafka-operator/blob/d8ef6afa8968fb02213afdf86e3c60f91def87a0/pkg/resources/kafka/configmap.go#L122
I think we should modify it to return `return fmt.Sprintf("log.dirs=%s\n", strings.Join(mountPaths, ","))`

Comment (bernardoVale) [reply pkg/resources/kafka/configmap.go]: that was the original implementation. I decided to use the `if` statement because if you don't declare a `storageConfig` you will end up with an empty `log.dirs=` declaration.

If you think that's not a problem a can fix that. We'll also need to fix all tests to include the empty `log.dist=`

Comment (baluchicken) [reply pkg/resources/kafka/configmap.go]: Thanks for the clarification, I get your point and you are right.
